server/src/routes/appointment.js

```diff
@@ -32,52 +54,57 @@ router.post('/:buyer', (req, res) => {
     let provider = '';
 
     Service.findById(req.body.service_id).then(service => {
-        if (service) provider = service.provider;
-        else
+        if (service) {
```

Comment:
this is the same code as right below just formatting, yes?

Reply:
Not really formatting, more like restructuring. But yes